<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartPage</class>
 <widget class="QWidget" name="CartPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文宋体</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #f5af19;  /* fallback for old browsers */
background:qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:0,stop:0 #f5af19,stop:1 #f12711);
color:rgb(255, 255, 255);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>华文琥珀</family>
          <pointsize>21</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 21pt &quot;华文琥珀&quot;;</string>
        </property>
        <property name="text">
         <string>购 物 车</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_item"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lab_tolprice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Adobe 黑体 Std R&quot;;</string>
        </property>
        <property name="text">
         <string>总价:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_buy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(255, 85, 0);                
border:1px solid;                  
border-radius:10px;
border-color:rgba(255,255,255,30);
font: 14pt &quot;黑体&quot;;
color:white;
padding:5px;
}
QPushButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton:hover{
background-color:rgb(255, 172, 5);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}</string>
        </property>
        <property name="text">
         <string>结算</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
